Adding link to the CNCF sandbox submission (#6816)
radius-project/radius: README.md

# Radius

> Radius is a cloud-native application platform that enables developers and the platform engineers that support them to collaborate on delivering and managing cloud-native applications that follow organizational best practices for cost, operations and security, by default. Radius is an open-source project that supports deploying applications across private cloud, Microsoft Azure, and Amazon Web Services, with more cloud providers to come. We have submitted Radius for consideration as a Cloud Native Computing Foundation (CNCF) [Sandbox project](https://github.com/cncf/sandbox/issues/65).

Reviewer: Instead of adding this to the end of the intro paragraph let's break this out into it's own section. In the future we could have a dedicated "Governance" section but for now a single sentence will work:

```suggestion
Radius is a cloud-native application platform that enables developers and the platform engineers that support them to collaborate on delivering and managing cloud-native applications that follow organizational best practices for cost, operations and security, by default. Radius is an open-source project that supports deploying applications across private cloud, Microsoft Azure, and Amazon Web Services, with more cloud providers to come.

Radius has been submitted to the Cloud Native Computing Foundation (CNCF) as a [Sandbox project](https://github.com/cncf/sandbox/issues/65).
```

Author: Sounds good! Add this as own section and retaining the language.
